src/view/com/post-thread/PostThreadItem.tsx

```diff
@@ -753,10 +753,9 @@ function ExpandedPostDetails({
             </Text>
 
             <InlineLinkText
-              to="#"
```

Comment:
should this just be `to={translatorUrl}`? any reason not to do that?

Reply:
Good point! Will update